app/models/espinita/audit.rb

```diff
@@ -24,6 +24,12 @@ def ancestors
         auditable_id, auditable_type, version])
     end
 
+    # Return all audits newer than current one (used for rollback)
+    def descendants
+      self.class.where(['auditable_id = ? and auditable_type = ? and version >= ?',
```

Comment:
Can we leverage Adel table methods here instead of a string to reduce complications in case the query needs to be extended? For example
Self.class.arel_table[:auditable_id].eq(auditable_id).and(arel_table[:auditable_type].eq(auditable_type).and(arel_table[:version].lt_eq(version)

Not 100% accurate but you get the idea. It ensures maximum compatability.